src/frontends/pytorch/src/frontend.cpp

```diff
@@ -38,16 +38,24 @@ namespace frontend {
 namespace pytorch {
 
 namespace {
-std::set<std::string> get_unconverted_types_from_model(const std::shared_ptr<Model>& model) {
-    std::set<std::string> unconverted_ops_types;
+std::map<std::string, std::string> get_unconverted_types_from_model(const std::shared_ptr<Model>& model) {
+    std::map<std::string, std::string> unconverted_ops_types;
     for (const auto& node : model->get_ordered_ops()) {
         if (const auto& fw_node = ov::as_type_ptr<PtFrameworkNode>(node)) {
-            auto op_type = fw_node->get_decoder()->get_op_type();
-            unconverted_ops_types.insert(op_type);
+            const auto& attrs = fw_node->get_attrs();
+            FRONT_END_GENERAL_CHECK(attrs.find(PtFrameworkNode::op_type_key) != attrs.end(),
+                                    "FrameworkNode attributes do not contain operation type.");
+            std::string exception_msg;
+            if (attrs.find(PtFrameworkNode::failed_conversion_key) != attrs.end()) {
+                exception_msg = attrs.at(PtFrameworkNode::failed_conversion_key);
+            }
+            if (!unconverted_ops_types.count(attrs.at(PtFrameworkNode::op_type_key))) {
+                unconverted_ops_types[attrs.at(PtFrameworkNode::op_type_key)] = exception_msg;
+            }
         }
         if (const auto& fw_node = ov::as_type_ptr<ov::op::util::MultiSubGraphOp>(node)) {
             for (size_t i = 0; i < fw_node->get_internal_subgraphs_size(); i++) {
-                auto internal_types = get_unconverted_types_from_model(fw_node->get_function(i));
+                const auto& internal_types = get_unconverted_types_from_model(fw_node->get_function(i));
                 unconverted_ops_types.insert(internal_types.begin(), internal_types.end());
             }
         }
```

Comment:
can we re-use the same function from TF FE? And move it to the common part?

Reply:
Both functions have slightly different approach, unification can be done as next stage, ad will require changing TF FE